src/pages/index.js

```diff
@@ -7,55 +7,78 @@ import UserInfo from "../components/UserInfo.js";
-  const formValidator = new FormValidator(config, form);
-  formValidator.enableValidation();
-});
+import Api from "../components/Api.js";
 
 const editButton = document.querySelector(".profile__edit-button");
-const editProfileModal = document.querySelector("#edit-modal");
-const profileTitle = document.querySelector(".profile__title");
-const profileSubtitle = document.querySelector(".profile__subtitle");
+
 const inputTitle = document.querySelector(".modal__input-title");
 const inputSubtitle = document.querySelector(".modal__input-subtitle");
-const profileCloseButton = document.querySelector(".modal__close-button");
-const profileEditForm = editProfileModal.querySelector(".modal__container");
-const cardList = document.querySelector(".cards__list");
-const cardTemplate = document.querySelector("#card-template");
 const addButton = document.querySelector(".profile__add-button");
 const addCardModal = document.querySelector("#modal-add-card");
-const closeAddFormButton = addCardModal.querySelector(".modal__close-button");
 const addCardForm = addCardModal.querySelector("#add-card-form");
-const cardName = addCardModal.querySelector(".modal__input-name");
-const cardLink = addCardModal.querySelector(".modal__input-link");
-const previewImageModal = document.querySelector("#modal-preview");
-const closePreviewButton = previewImageModal.querySelector(
-  ".modal__close-button"
-);
-const previewImageElement = previewImageModal.querySelector(
-  ".modal__preview-image"
-);
-const previewImageElementName = previewImageModal.querySelector(
-  ".modal__image-title"
+const profileAvatarContainer = document.querySelector(
+  ".profile__image-overlay"
 );
 
+// get all form elements
+//const forms = document.querySelectorAll(config.formSelector);
+//console.log(forms);
+
+// initialize FormValidator for each form
+
+const formValidators = {};
+
+const enableValidation = (config) => {
+  const formList = Array.from(document.querySelectorAll(config.formSelector));
+  formList.forEach((formElement) => {
+    const validator = new FormValidator(config, formElement);
+    const formName = formElement.getAttribute("name");
+
+    formValidators[formName] = validator;
+    validator.enableValidation();
+  });
+};
+
+enableValidation(config);
+
+// API
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "6bc22e8b-f205-4c04-b4ee-f39ef81e1654",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getUserInfo()
+  .then((data) => {
+    userInfo.setUserInfo(data);
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+const handleAvatarSubmit = ({ avatar }) => {
+  console.log(avatar);
+  editAvatarPopup.renderLoading(true);
+  api
+    .updateAvatar(avatar)
+    .then((data) => {
+      userInfo.setUserInfo(data);
+      editAvatarPopup.close();
+      formValidators["edit-profile-form"].resetValidation();
+      formValidators["edit-profile-form"].clearFormFields();
```

Comment:
`edit-profile-form` is for the profile form but here you submit the **avatar** form